run_cibuildwheel_on_ec2.sh

```diff
@@ -0,0 +1,32 @@
+if [ $# -ne 6 ]; then
```

Comment:
do we want this piece of code stay in the repo? I know it is operationally needed but does it really make sense to a collaborator? If yes, is this at the right format/structure currently?

Reply:
This scrip could be used by collaborator to build wheels and run integration tests inside their EC2 instance. What we can improve here is to provide later some instruction on how to use it.